crates/core/database/src/models/admin_migrations/ops/mongodb/scripts.rs

```diff
@@ -26,7 +26,7 @@ pub const LATEST_REVISION: i32 = 31;
 pub async fn migrate_database(db: &MongoDb) {
     let migrations = db.col::<Document>("migrations");
     let data = migrations
-        .find_one(None, None)
+        .find_one(Document::new())
```

Comment:
Maybe worth using `doc! {}` here for consistency?
I wonder if there is a Default::default() implementation...